/**:
  ros__parameters:
    object_primitives: "1,1,3"
    object_dimensions:
      - "1.4,1.5,0.2"
      - "0.45,0.05,0.6"
      - "0.6,0.05,0.0"
    # poses xyz wxyz
    object_poses:
      - "1.0,0.0,-0.5,1.0,0.0,0.0,0.0"
      - "0.4,0.1,0.06,1.0,0.0,0.0,0.0"
      - "0.9,0.1,0.3,1.0,0.0,0.0,0.0"